src/test/java/core/basesyntax/ReportWriterServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import core.basesyntax.service.ReportWriterService;
+import core.basesyntax.service.impl.ReportWriterServiceImpl;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportWriterServiceTest {
+    private static final List<String> REPORT =
+            List.of("fruit,quantity", "banana,20", "apple,100");
+    private static final String INVALID_PATH = "C:/Users/Bob/project/src/main/resources/file.csv\"";
+    private static final String PATH_TO_ORIGINAL_REPORT_FILE = "src/test/resources/report.csv";
+    private static final String PATH_TO_TEST_REPORT_FILE = "src/test/resources/testReport.csv";
+    private static ReportWriterService reportWriterService;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        reportWriterService = new ReportWriterServiceImpl();
+        Files.write(Path.of(PATH_TO_ORIGINAL_REPORT_FILE), ("fruit,quantity\n"
+                + "banana,20\n" + "apple,100\n" + "orange,10\n").getBytes(StandardCharsets.UTF_8));
+    }
+
+    @Test
+    public void saveReport_ok() {
```

Comment:
```suggestion
    public void saveReport_ok() {
```
make all test method names consistent